### PR TITLE
Ifrost/207515 storybook cleanup

### PR DESCRIPTION
**Pull request by ifrost1**
### Summary:
- PLEASE add your own well-thought-out language around this since you are much brighter and articulate than I am.
- Reorganize stories within Examples to get rid of that category (from plop template)
- Reorganize pages to align with what product teams are calling these
- Add Getting Started. Make that the homepage

### Test Plan:
`yarn types` runs
`yarn test` runs
`yarn lint` runs
`yarn start` runs

### CODE REVIEW

File: docs/GETTING_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+EDS' Storybook structure follows the [Atomic Design](https://bradfrost.com/blog/post/atomic-web-design/) methodology where smaller components work together to create larger components to create templates and pages. To navigate, you can expand the items in the left navigation or perform a search for what you are looking for.
+
+The navigation consists of:
+
+- **LegacyTokens** contains legacy tokens before the system was refactored [TODO]
+- **LegacyTypographySizes** contains legacy typography sizes before the system was refactored [TODO]
+- **DesignTokens** includes colors, typography, and other properties that build a theme across the system. These are shipped with the system to be consumed by product teams.
+- **Documentation** contains documenation around code guidelines, component structure, icon, layout, token, and typography
+- **Atoms**are the smallest level of components within the system. These are shipped with the system to be consumed by product teams.
```
Comment (dierat) [inline docs/GETTING_STARTED.md:19]:
```suggestion
- **Atoms** are the smallest level of components within the system. These are shipped with the system to be consumed by product teams.
```

**Comment by dierat** [comment]
Looking at storybook on your branch, I'm seeing `StackedBlock` still in the `Example` section.

**Comment by dierat** [comment]
I'm thinking we should also update the plop files and examples in the documentation. Maybe instead of `Example` we could use something really overt like `PleaseChangeThisToADifferentFolder`.

**Comment by ifrost1** [comment]
@dierat Those changes were made. Thank you!

**Pull request merged**